ils_middleware/tasks/sinopia/local_metadata.py

```diff
@@ -79,10 +78,13 @@ def new_local_admin_metadata(*args, **kwargs):
     sinopia_api_uri = Variable.get("sinopia_api_uri")
 
     for resource_uri in resources:
-        resource = ast.literal_eval(
-            task_instance.xcom_pull(key=resource_uri, task_ids="sqs-message-parse")
+        message = task_instance.xcom_pull(
+            key=resource_uri, task_ids="sqs-message-parse"
         )
+        resource = message.get("resource")
         group = resource.get("group")
+        logger.debug(resource)
```

Comment:
Do we want to keep `debug` loggers? I'm fine either way.

Reply:
Good question, the debug logs only when we're running locally with Docker as the `docker-compose` and isn't set when deploying to AWS. Maybe a discussion topic?

Reply:
That's a good point, maybe keep them in place since it would allow us to turn debug logging on if necessary and not have to change code.